packages/cli-e2e/__tests__/react.specs.ts

```diff
@@ -87,35 +94,24 @@ describe('ui:create:react', () => {
 
   const startApplication = async (
     processManager: ProcessManager,
-    debugName = 'react-server',
-    errorCallback: (error: string) => void = () => {}
+    debugName = 'react-server'
   ) => {
+    const args = ['npm', 'run', 'start'];
+    if (process.platform === 'win32') {
+      args[0] = npmJsPath();
+      args.unshift('node');
```

Comment:
This seems very similar to above, where we need to shift the "node" arg for windows.

Could benefit from a `makeArgsPlatformAgnostic(args)` kinda utility function ?

Reply:
Yeah, that's something I thought about.
I'd like to revisit this in the future because I'm unhappy with how I handle that (i.e. replicating what `npm.cmd` does in the terminal to remove the node layer, which lessens the 'end-to-end' aspect.)

I'll get a look tho to refactor this.